lms/static/js/jquery.gradebook.js

```diff
@@ -1,24 +1,19 @@
-
-
-
 var Gradebook = function($element) {
-	var _this = this;
-	var $element = $element;
+  "use strict";
 	var $body = $('body');
 	var $grades = $element.find('.grades');
 	var $studentTable = $element.find('.student-table');
 	var $gradeTable = $element.find('.grade-table');
 	var $search = $element.find('.student-search-field');
 	var $leftShadow = $('<div class="left-shadow"></div>');
-	var $rightShadow = $('<div class="right-shadow"></div>');
```

Comment:
I removed right shadow completely because when I scroll to right it inserts an empty cell on right most side (issue only in scrolling not in dragging)